op-deployer/pkg/deployer/verify/artifacts.go

```diff
@@ -0,0 +1,97 @@
+package verify
+
+import (
+	"encoding/json"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/foundry"
+)
+
+type contractArtifact struct {
```

Comment:
is there an existing struct for this (e.g. in op-chain-ops)? IIRC we've build similar structs elsewhere. no worries if one doesn't exist but would like to reuse what we already have if possible

Reply:
There's a much more extensive version of this struct in `op-chain-ops` called `foundry.Artifact`. But it contains many fields we will ignore. The nice thing about this additional struct is that it is minimal and only contains fields that get used during contract verification. I'd prefer to keep this one instead of having to dive deep into the `foundry.Artifact` in all of the downstream functions to pull out the fields we actually need for verification